bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TT_TW_PHOTOPRODUCTION/TW_PHOTOPRODUCTION_NEGATIVE/antoniocotar_tw_NLO_photoproduction_negative_madspin_card.dat

```diff
@@ -0,0 +1,16 @@
+#Some options (uncomment to apply)
+#
+# set seed 1
+# set Nevents_for_max_weight 75 # number of events for the estimate of the max. weight
```

Comment:
for MG version 26X, "Nevents_for_max_weight" should be "Nevents_for_max_weigth"

Reply:
I need to uncomment this line?

Reply:
the difference is "ht" and "th"

Reply:
Ok, I applied that change, but I had a doubt whether I needed to uncomment that line. Thank you